sktime/performance_metrics/forecasting/_coerce.py

```diff
@@ -42,3 +42,66 @@ def _coerce_to_1d_numpy(obj):
     if isinstance(obj, (pd.DataFrame, pd.Series)):
         obj = obj.values
     return obj.flatten()
+
+
+# A tiny “callable → Metric” wrapper, in case the user passed a bare function
+class _CallableForecastingErrorMetric:
```

Comment:
this looks like the `_DynamicForecastingErrorMetric`, just without the init fields